### ISSUE TITLE
Koboldite.

### ISSUE THREAD

**Issue by miner-bob-1960** [MyM-ModpackTeam/hypovolemia#356]
is there a way to get Hobgolins to spawn in the 1.8 update or a way to get Kobldite, as its needed for infinity ingots.

**Comment by chuggafan** [comment]
Large pastures spawn them over time iirc

**Comment by miner-bob-1960** [comment]
thay spawn in forest and some times plains biomes, and there is no way to change the biomes as none of the biome changing works.

**Comment by chuggafan** [comment]
IIRC the witchery biome changing still works, try it

**Comment by Messiah7** [comment]
Yep, witchery rite of shifting seasons work fine, i change biome to forest and growing some trees, but no one hobgoblin for 5 hours (biome like a circle ~5x5 chunks)
